chunks/generative_models.ipynb

```diff
@@ -0,0 +1,397 @@
+    }
+   },
+   "source": [
+    "## PGM interpretation\n",
+    "\n",
+    "* Following the arrows, the network of conditional dependences shows you how to go about simulating mock data ($N_k$)\n",
+    "\n",
+    "> For example: \n",
+    "1. Draw a sample $\\theta$ vector;\n",
+    "2. Compute $\\mu_k$ from it;\n",
+    "3. Draw an $N_k$ given that $\\mu_k$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## PGM interpretation\n",
+    "\n",
+    "* The graph is also an illustration of a particular factorisation of a joint probability distribution: the PDF for every variable in the model \n",
+    "\n",
+    "* For example: \n",
+    "\n",
+    "$P(N_k|\\mu_k,H)\\;P(\\mu_k|\\theta_k,H)\\;P(\\theta_k|H) =  P(N_k,\\mu_k,\\theta_k|H)$\n",
+    "\n",
+    "> Note that the dependence of $N_k$ on $\\theta$ has been dropped here: $N_k$ is only dependent on $\\theta$ through $\\mu_k$. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Simulating the whole image\n",
+    "\n",
+    "* Our model for the cluster allows us to predict all of the image pixel values, separately: each of the $\\mu_k$ depends on $\\theta$ alone\n",
+    "\n",
+    "* Our simple Poisson model reflects an assumption about our detector, which is that the observed counts in the $k^{th}$ pixel are independent of the observed counts in all the other pixels. $N_k$ only depends on its pixel's expected counts $\\mu_k$\n",
+    "\n",
+    "* In this case, the sampling distribution for all the observed pixel values $\\boldsymbol{N}$ factorizes and simplifies to\n",
+    "\n",
+    "$\\;\\;\\;\\;\\;\\;\\;\\;\\;\\;P(\\boldsymbol{N}|\\boldsymbol{\\mu},H) = \\prod_k P(N_k|\\mu_k,H)$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## All pixels, fixed inputs\n",
+    "\n",
+    "<img src=\"../graphics/pgms_all_pixels_input_fixed.png\">\n",
+    "\n",
+    "$\\;\\;\\;\\;\\;\\;\\;P(\\boldsymbol{N},\\boldsymbol{\\mu},\\theta|H) = \\left[ \\prod_k P(N_k|\\mu_k,H)  P(\\mu_k|\\theta,H) \\right] P(\\theta|H)$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## PGM pieces\n",
+    "\n",
+    "* Each _node_ in the graph represents a PDF, for the variable labeled inside it\n",
+    "\n",
+    "* Each _edge_ in the graph represents a conditional dependence\n",
+    "\n",
+    "* Deterministic relationships are indicated by \"fixed\" variables represented by solid points\n",
+    "\n",
+    "* Plates contain conditionally independent variables\n",
+    "\n",
+    "> Think of plates as illustrating a stack of layers, seen from above, that are only connected by their edges to variables outside the plate."
```

Comment:
"edges" here is potentially confusing, since the plate rectangle also has edges, but I can't think of an improvement (other than maybe "edges (arrows)")